releng/MWE.setup

```diff
@@ -302,7 +302,7 @@
           name="org.eclipse.pde.feature.group"/>
       <requirement
           name="com.google.guava"
-          versionRange="[27.1.0,28.0.0)"/>
+          versionRange="[30.1.0,31.0.0)"/>
```

Comment:
would expect and orbit url update here too

Reply:
I have just updated the orbit url from `https://download.eclipse.org/tools/orbit/downloads/2019-12` to `https://download.eclipse.org/tools/orbit/downloads/2021-03` also in the `MWE.setup` file

Reply:
guess we should then have secsions for the 2020er and the 2021-03 releases the the file then